app-misc/bgrep/bgrep-1_p20210321.ebuild

```diff
@@ -0,0 +1,43 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit toolchain-funcs
+
+DESCRIPTION="Grep-like tool to search for binary strings"
+HOMEPAGE="https://github.com/tmbinc/bgrep/"
+SRC_URI="https://github.com/tmbinc/bgrep/archive/master.tar.gz -> ${P}.tar.gz"
```

Comment:
This isn't pinned to a commit, so the hash is going to change on new commits. You want to fetch a particular commit from that day.

Reply:
Resolved